posthog/test/test_decorators.py

```diff
@@ -37,8 +35,13 @@ def setUp(self) -> None:
         super().setUp()
 
     def test_returns_fresh_result(self) -> None:
+        router.register(r"dummy", DummyViewSet, "dummy")
+        self.client = Client()
+        self.client.force_login(self.user)
         response = self.client.get(f"/api/dummy").json()
 
+        print(response)  # noqa: T201
```

Comment:
:O

Reply:
this is me trying to debug a CI only failure 😓  - still doesn't work